.github/workflows/apply-applications-sbx.yaml

```diff
@@ -0,0 +1,28 @@
+name: Deploy to the sbx environment
+
+on:
+  push:
+    branches:
+      - 'main'
+
+jobs:
+
+  deploy:
+
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Git checkout
+        uses: actions/checkout@v1
+
+      - uses: tale/kubectl-action@v1
+        with:
+          base64-kube-config: ${{ secrets.KUBE_CONFIG_DEV }}
```

Comment:
KUBE_CONFIG_SBX

you should create a service account with cluster-admin permissions, then generate kube-config.yaml and create a GitHub secret KUBE_CONFIG_SBX containing its content (base64 encoded)

Reply:
Renamed

Reply:
I have also created a service account with cluster-admin permissions, then generated kube-config.yaml file which, according to our discussion via email, has been sent to you for GitHub secret creation.